spec/models/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should have_valid(:first_name).when("Jimmy", "Linc")}
+  it { should_not have_valid(:first_name).when(nil, "")}
+
+  it { should have_valid(:last_name).when("Ranger", "Kupke")}
+  it { should_not have_valid(:last_name).when(nil, "")}
+
+  it { should have_valid(:email).when("example@gmail.com", "erlich@aviato.net")}
+  it { should_not have_valid(:email).when("example", "example@", "exam@gma", "example.com", nil)}
+
+  describe "#admin?" do
+    context "when user is a admin" do
+      it "should give back true if user an admin" do
+        user = create(:user, role: "admin")
+        expect(user.admin?).to eq(true)
+      end
+    end
+    context "when user is a member" do
+      it 'should give back false' do
```

Comment:
If you want to use single quotes throughout, you can change the yml file so hound will stop throwing comments in there. That said, try to stick with one or the other, either using single as the default or double as the default